tests/opentelemetry-docker-tests/tests/docker-compose.yml

```diff
@@ -10,4 +10,5 @@ services:
   otcollector:
     image: otel/opentelemetry-collector:0.22.0
     ports:
-        - "4317:4317"
\ No newline at end of file
+        - "4317:4317"
+        - "55681:55681"
```

Comment:
Correct https://github.com/open-telemetry/opentelemetry-collector/blob/cda019f1a73fcd7528256ce88beafb3a8e30d664/receiver/otlpreceiver/factory.go#L34. This might change in future but for now this is the default http port.